aiotg/chat.py

```diff
@@ -63,6 +63,45 @@ def _send_to_chat(self, method, **options):
             **options
         )
 
+    def get_chat(self):
+        """
+        Get information about the chat.
+        """
+        return self.bot.api_call(
+            "getChat",
+            chat_id=str(self.id)
+        )
+
+    def get_chat_administrators(self):
+        """
+        Get a list of administrators in a chat. Chat must be not private.
+        """
+        return self.bot.api_call(
+            "getChatAdministrators",
+            chat_id=str(self.id)
+        )
+
+    def get_chat_members_count(self):
+        """
+        Get the number of members in a chat.
+        """
+        return self.bot.api_call(
+            "getChatMembersCount",
+            chat_id=str(self.id)
+        )
+
+    def get_chat_member(self, user_id):
+        """
+        Get information about a member of a chat.
+
+        :param user_id: int Unique identifier of the target user
+        """
+        return self.bot.api_call(
+            "getChatMember",
+            chat_id=str(self.id),
+            user_id=int(user_id)
```

Comment:
I don't have any problem with this code. But I've found  https://github.com/KeepSafe/aiohttp/issues/920 about integer. Ok. It would be safer to make str(user_id).